src/components/FileUpload.tsx

```diff
@@ -8,19 +8,35 @@ import axios from 'axios';
 import dynamic from 'next/dynamic';
 import IterationVisualizer from './IterationVisualizer';
 import { Shape } from './EditSVGPage';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from './ui/select';
 const EditSVGPage = dynamic(() => import('./EditSVGPage'), { ssr: false });
 
+type FabricUnit = 'in' | 'cm';
+
 const FileUpload = () => {
   const { toast } = useToast();
   const inputRef = React.useRef<HTMLInputElement>(null);
   const [filePath, setFilePath] = useState<string | null>(null);
   const [pdf, setPdf] = useState<string | Blob>('');
+  const [fabricLength, setFabricLength] = useState<number | undefined>(
```

Comment:
this should be fabric width actually

Reply:
yeah rename this var and then address any todos (not sure if you left them or someone else) and this PR should begood to go

Reply:
Fixed